bin/lib-install.sh

```diff
@@ -317,6 +380,8 @@ updateBashProfile() {
     if ! diff "$bashFile" "$new"; then
         log info "Updating: $bashFile"
         cat "$new" > "$bashFile"
+        log warn "Updated rc file: '$bashFile'" \
+            "You must launch a new shell or reload the rc file (\`source \"$bashFile\"\`) to see the changes in your \$PATH"
```

Comment:
not going to just open up a new shell for them?

Reply:
I'd prefer not to. When they exit out of that shell they'll be back in their original one without the directory in their $PATH so it'll be confusing.